src/generic_ui/locales/en/messages.json

```diff
@@ -667,6 +667,14 @@
     "description": "Notification for when the user has been logged out of the social network.",
     "message": "You have been logged out of __network__"
   },
+  "PRE_LOG_OUT": {
+  "description": "Notification to ask user if they are sure that they want to log out.",
+  "message": "Are you sure you want to log out? You are granting access to __name__ and __numOthers__ others"
+  },
+  "PRE_LOG_OUT_SIMPLE": {
+    "description": "Notification to ask user if they are sure that they want to log out.",
+    "message": "Are you sure you want to log out?"
```

Comment:
This message is supposed to be a check for when the user  isn't sharing access with anyone. It's just an "are you sure?" message. Should we take it out? Or perhaps give it a different name?